indico/modules/events/editing/operations.py

```diff
@@ -204,8 +211,25 @@ def create_submitter_revision(prev_revision, user, files):
     return new_revision
 
 
+@memoize
```

Comment:
Maybe we can add some extra flag to `EditingRevision` to simplify this check? Maybe whether the files were uploaded by the user or the editor?

Reply:
I really wanted to avoid migrations in this PR, specially since this can be achieved without storing extra data...
Maybe it would be cleaner to have this function as a property of `EditingRevision`? (although it's not used anywhere else other than this file)

Reply:
tbh i'd keep it here for now...